jsscripts/embedhelper.js

```diff
@@ -412,8 +417,13 @@ EmbedHelper.prototype = {
       case "DOMAutoComplete":
       case "blur": {
         let form = aEvent.target;
-        let win = form.ownerDocument.defaultView;
-        LoginManagerChild.forWindow(win).onFieldAutoComplete(form, null);
+        if (form.ownerDocument &&
+                ChromeUtils.getClassName(form) === "HTMLInputElement" &&
```

Comment:
I'm wondering about this condition. Won't this already be covered by [the check in `onFieldAutoComplete`](https://github.com/sailfishos-mirror/gecko-dev/blob/600df6aa643726626d2eab36bc0f6e34bbe5ac2a/toolkit/components/passwordmgr/LoginManagerChild.jsm#L1060)?

Reply:
`ChromeUtils.getClassName(acInputField.ownerDocument) != "HTMLDocument"`? That's checking the type of the ownerDocument, whereas this is checking the type of the element itself.

Reply:
Indeed, you're quite right, my mistake.